assets/src/edit-story/elements/image/index.js

```diff
@@ -30,6 +30,11 @@ export const defaultAttributes = {
   scale: 100,
   focalX: 50,
   focalY: 50,
+  flip: {
```

Comment:
IMHO this is also better to set up as a shared object geometry. The fact that we only expose it on a few element types is just the current UX choice.